samples/snippets/src/main/java/com/example/dialogflow/AnswerRecordManagement.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dialogflow;
+
+// [START dialogflow_update_answer_record]
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.dialogflow.v2.AnswerFeedback;
+import com.google.cloud.dialogflow.v2.AnswerRecord;
+import com.google.cloud.dialogflow.v2.AnswerRecordName;
+import com.google.cloud.dialogflow.v2.AnswerRecordsClient;
+import com.google.cloud.dialogflow.v2.UpdateAnswerRecordRequest;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+public class AnswerRecordManagement {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "my-location";
+
+    // Set id of the answer record to be updated.
+    // Answer records are created when there's a human agent assistant suggestion generated.
```

Comment:
`suggestion generated.` -> `suggestion is generated.`

Reply:
Thanks! Done